### PR TITLE
[NMS] Add subscriber JSON editor

### PR DESCRIPTION
**Pull request by HannaFar**
Signed-off-by: HannaFar <hannafarag159@gmail.com>

## Summary

Add subscriber JSON editor

Capture d’écran 2020-08-10 à 23 59 17
Capture d’écran 2020-08-10 à 23 59 29

### CODE REVIEW

File: nms/app/fbcnms-projects/magmalte/app/views/subscriber/SubscriberDetail.js [hunk 1]
```diff
@@ -147,6 +148,10 @@ export default function SubscriberDetail(props: {
         </Grid>
       </AppBar>
       <Switch>
+        <Route
+          path={relativePath('/config/json')}
```
Comment (karthiksubraveti) [inline nms/app/fbcnms-projects/magmalte/app/views/subscriber/SubscriberDetail.js]: is this per subscriber or for all subscribers ?

**Review by karthiksubraveti — COMMENTED** [review]
Can you post a screenshot or a screen recording of these changes. Thanks

File: nms/app/fbcnms-projects/magmalte/app/views/subscriber/SubscriberDetail.js (hunk 1 above)
Comment (HannaFar) [reply nms/app/fbcnms-projects/magmalte/app/views/subscriber/SubscriberDetail.js]: Per subscriber

Comment (karthiksubraveti) [reply nms/app/fbcnms-projects/magmalte/app/views/subscriber/SubscriberDetail.js]: if this per subscriber, shouldn't we have a subscriber id in the path ?

Comment (HannaFar) [reply nms/app/fbcnms-projects/magmalte/app/views/subscriber/SubscriberDetail.js]: It's already in the path

Comment (HannaFar) [reply nms/app/fbcnms-projects/magmalte/app/views/subscriber/SubscriberDetail.js]: We add it here: https://github.com/magma/magma/blob/master/nms/app/fbcnms-projects/magmalte/app/views/subscriber/SubscriberOverview.js#L127

**Review by karthiksubraveti — APPROVED** [review]

**Pull request merged**